cmd/influx_inspect/export.go

```diff
@@ -84,29 +90,65 @@ func (c *cmdExport) run() error {
 }
 
 func (c *cmdExport) export() error {
-	if err := c.walkFiles(); err != nil {
+	if err := c.walkTSMFiles(); err != nil {
+		return err
+	}
+	if err := c.walkWALFiles(); err != nil {
 		return err
 	}
 	return c.writeFiles()
 }
 
-func (c *cmdExport) walkFiles() error {
-	err := filepath.Walk(c.path, func(path string, f os.FileInfo, err error) error {
+func (c *cmdExport) walkTSMFiles() error {
+	err := filepath.Walk(c.dataDir, func(dataDir string, f os.FileInfo, err error) error {
 		if err != nil {
 			return err
 		}
-		if filepath.Ext(path) == c.ext {
-			//files = append(files, path)
-			relPath, _ := filepath.Rel(c.path, path)
+
+		ext := fmt.Sprintf(".%s", tsm1.TSMFileExtension)
+		if filepath.Ext(dataDir) == ext {
+			relPath, _ := filepath.Rel(c.dataDir, dataDir)
 			dirs := strings.Split(relPath, string(byte(os.PathSeparator)))
 			if dirs[0] == c.db || c.db == "" {
 				if dirs[1] == c.retentionPolicy || c.retentionPolicy == "" {
 					key := filepath.Join(dirs[0], dirs[1])
-					files := c.files[key]
+					files := c.tsmFiles[key]
 					if files == nil {
 						files = []string{}
 					}
-					c.files[key] = append(files, path)
+					c.manifest[key] = struct{}{}
+					c.tsmFiles[key] = append(files, dataDir)
+				}
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+	return nil
+}
+
+func (c *cmdExport) walkWALFiles() error {
```

Comment:
Even though you didn't introduce these changes, this method could benefit from the `error` tidy up too.